FluidNC/src/Maslow/Maslow.h

```diff
@@ -41,15 +45,32 @@ class Maslow_ {
     void takeMeasurement(float lengths[]);
     void moveWithSlack(float x, float y, bool leftBelt, bool rightBelt);
     void takeUpInternalSlack();
-    float computeVertical(float firstUpper, float firstLower, float secondUpper, float secondLower);
-    void computeFrameDimensions(float lengthsSet1[], float lengthsSet2[], float machineDimensions[]);
-    void retractBR();
+    void retractBR_CAL();
+    void retractBL_CAL();
+    void stopMotors();
+
+    void retractTL();
+    void retractTR();
     void retractBL();
+    void retractBR();
+    void retractALL();
+    void extendALL();
+    void comply();
+    void stop();
+    bool retractingTL = false;
```

Comment:
We're starting to have a lot of flags (my fault) and I wonder if there is a better way to organize them?